eppo/src/main/java/cloud/eppo/android/util/Utils.java

```diff
@@ -59,4 +61,17 @@ public static String getISODate(Date date) {
     public static SharedPreferences getSharedPrefs(Context context) {
         return context.getSharedPreferences("eppo", Context.MODE_PRIVATE);
     }
+
+    public static String generateExperimentKey(String featureFlagKey, String allocationKey) {
+        String encodedAllocationKey = "";
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            encodedAllocationKey = Base64.getEncoder().encodeToString(allocationKey.getBytes());
+        } else {
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.FROYO) {
+                encodedAllocationKey =  android.util.Base64.encodeToString(allocationKey.getBytes(), android.util.Base64.DEFAULT);
+            }
+        }
```

Comment:
I hope no
body is using android version less than 8 😓

Reply:
Yes, removed it. Now we don't need to base encode. So, simply sending `<featureKey>-<allocationKey>`